eng/UpdateAzureSdkCodes.ps1

```diff
@@ -0,0 +1,12 @@
+param([string]$Branch, [string]$SdkRepoRoot, [string[]]$ServiceDirectoryFilters)
+
+Write-Host "Generating Azure SDK Codes..."
+foreach ($filter in $ServiceDirectoryFilters) {
+    Write-Host 'Generating projects under service directory ' -ForegroundColor Green -NoNewline
+    Write-Host "$filter" -ForegroundColor Yellow
+    dotnet msbuild /restore /t:GenerateCode /p:ServiceDirectory=$filter "$SdkRepoRoot\eng\service.proj"
+    if ($LastExitCode -ne 0) {
+        Write-Error 'Generation error'
```

Comment:
is there a way that we could emphasize the lines that reports the errors on the script finishes?

Reply:
It's a known issue (or by design) that `dotnet msbuild` is different from `msbuild`. You need to either set the verbose level to normal (default is minimal) or use a file logger. https://github.com/dotnet/msbuild/issues/4288

Reply:
well at least we have a clear information to tell us it failed. Previously it just outputs nothing

Reply:
I only enabled it for CI, since we split the build. Each job will not print too many logs.